react/src/lib/components/DeckGLMap/layers/terrain/map3DLayer.ts

```diff
@@ -0,0 +1,73 @@
+import { CompositeLayer } from "@deck.gl/core";
+import TerrainMapLayer, {
+    TerrainMapLayerProps,
+    TerrainMapLayerData,
+} from "./terrainMapLayer";
+import { ExtendedLayerProps } from "../utils/layerTools";
+import { layersDefaultProps } from "../layersDefaultProps";
+import { TerrainLoader } from "@loaders.gl/terrain";
```

Comment:
Got a typescript error from this line. Replacing with 
```
const terrain = require("@loaders.gl/terrain");
```
seems to work.

@shadab-skhan @shruthirai @Vladimir-Kokin any of you get an error?

Reply:
I think you have to install it.. npm install loaders.gl/terrain